.npmrc

```diff
@@ -0,0 +1 @@
+v16.0.0
```

Comment:
The format is invalid, as it needs to be something like `key = value`.

Also what's the intention here? To only use node version 16? I don't think this is necessary given that the publish workflow specifies the version, and would prefer for this file to be removed.

Reply:
I added to make sure devs used 16 while changing `package-lock.json`, but I'm happy to remove it.